src/main/java/com/iexec/sms/secret/teetaskruntime/AppRuntimeSecretController.java

```diff
@@ -147,6 +152,95 @@ public AppRuntimeSecretController(AuthorizationService authorizationService,
 
         return ResponseEntity.noContent().build();
     }
+    // endregion
+
+    // region App requester endpoint
+    @PostMapping("/requesters/{requesterAddress}/apps/{appAddress}/secrets/{secretIndex}")
+    public ResponseEntity<String> addAppRequesterAppRuntimeSecret(@RequestHeader("Authorization") String authorization,
```

Comment:
You can update role name everywhere to ~~AppRequester~~ Requester.

Reply:
Sure, I'll do it in the last PR: https://github.com/iExecBlockchainComputing/iexec-sms/pull/76

Reply:
Done there: https://github.com/iExecBlockchainComputing/iexec-sms/commit/a6e17fd41d18dda94b9133dbb34ecbbb0696c336